src/containers/ConnectorManagement.jsx

```diff
@@ -72,7 +72,7 @@ export default class ConnectorManagement extends Component {
 
   render () {
     const { accounts } = this.state.connector
-    const { isConnected, selectedAccount, isWorking } = this.state
+    const { isConnected, selectedAccount, isWorking, jobInBackground } = this.state
```

Comment:
I realise that `jobInBackground` and `backgroundWorking` might be confusing as its a working server side… On #150, I choosed to call it `hadSuccessTimeout`.